packages/lib-react-components/src/Modal/components/ModalHeading/ModalHeading.js

```diff
@@ -43,16 +43,12 @@ function ModalHeading ({ background = 'brand', color = 'neutral-6', className =
   )
 }
 
-ModalHeading.defaultProps = {
-  closeFn: undefined
-}
```

Comment:
I didn't replace this with a default param in the destructured `props` because there's a conditional feature in ModalHeading where the CloseButton is rendered only if `closeFn` is defined.